data/Karmpatel960.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Karm Patel",
+  "type": "personal",
+  "bio": "Open Source Enthusiast | DevOps | Learning New tech | Learning in Pubic"
```

Comment:
Missing a command

```suggestion
  "bio": "Open Source Enthusiast | DevOps | Learning New tech | Learning in Pubic",
```

Reply:
@Karmpatel960 please also take a look at the misspelling of the world `Public`